src/Dependencies/Threading/TestHooks/IAsyncToken.cs

```diff
@@ -10,4 +10,5 @@ namespace Microsoft.CodeAnalysis.Shared.TestHooks;
 
 internal interface IAsyncToken : IDisposable
 {
+    bool IsNull { get; }
```

Comment:
You refer to it as "empty" tokens. Would `IsEmpty` make more sense here? Also, do we plan to update EmptyAsyncToken to return `true`?

Reply:
> Also, do we plan to update EmptyAsyncToken to return true?

Yes, forgot to push the commit.